### DISCUSSION TITLE
parameter K in p2g analysis

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1316] [Questions / Documentation] [answered]
Hi @rcorces,

I was wondering if it does make sense to play with parameter k (The number of k-nearest neighbors to use for creating single-cell groups for correlation analyses.) for the peak to gene correlation analysis? Or I should avoid that if one of my groups has a low number of cells? (but overall, I have 90,000 cells)

Thanks again

**Comment by rcorces** [comment, marked as answer]
I'm not sure I have a concrete answer. If k is too low, there wont be enough signal in the cell group. If k is too high, you might end up missing signal from rare cell types/states. So the choice of k is really related to the biology of your system. I wouldnt set k lower than 50, primarily based on the fact that I wouldnt trust a "cluster" or group of cells that had fewer than 50 cells in it as being something biologically meaningful.

**Reply by rojinsafavi** [reply]
Thanks @rcorces !
